### PR TITLE
Basic questions of greedy algorithm

### PR DESCRIPTION
**Pull request by akratikushwah**
<hr>

## Description 📜
issue number #6444 
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #issue_number_here 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [X] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [X] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [X] I have performed a self-review of my own code or work.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] My changes generates no new warnings.
- [X] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: C_CPP/Algorithmic Approaches/Greedy/readme.md [hunk 1]
```diff
@@ -1 +1 @@
-
+[Greedy Algorithm basic questions](https://github.com/girlscript/winter-of-contributing/pull/6803/files/b76057d494478b85e66a523c363a844bc845b34f#diff-b9433e4f3ca2bd6a9d5ce791c5abd3f0003ff25b3d1a162acbb53b045af4b181)
```
Comment (vasu-1) [inline C_CPP/Algorithmic Approaches/Greedy/readme.md:1]:
```suggestion
[Greedy Algorithm basic questions](Greedy%20algorithms%20questions.md)
```

**Review by vasu-1 — CHANGES_REQUESTED** [review]
add a correct file link!

**Comment by akratikushwah** [comment]
please check again @vasu-1

**Comment by akratikushwah** [comment]
This PR is not visible on my profile @vasu-1 @Saviour1001

**Pull request merged**